ted_sws/mapping_suite_processor/entrypoints/cli/cmd_mapping_suite_processor.py

```diff
@@ -50,68 +86,154 @@ def __init__(
             raise FileNotFoundError(error_msg)
 
     def _cmd(self, cmd: str):
-        if cmd == 'normalisation_resource_generator':
-            cmd_generate_mapping_resources.run(
+        if cmd == 'resources_injector':
+            cmd_resources_injector.run(
                 mapping_suite_id=self.mapping_suite_id,
                 opt_mappings_folder=self.mappings_path
             )
-        elif cmd == 'resources_injector':
-            cmd_resources_injector.run(
+        elif cmd == 'rml_modules_injector':
+            cmd_rml_modules_injector.run(
```

Comment:
I recommend to use Chain of Responsibility Design Pattern instead of else-if construction.

Reply:
not the case here. 
here is just a pick (as python does not have switch-case statement)
"chain of responsibility" is about more complex problems, involving a lot of unneeded code (more java-like).